config/routes.rb

```diff
@@ -32,6 +32,8 @@
       get :edit
       get '/speaker-emails' => 'events#speaker_emails', as: :speaker_email_notifications
       get '/guidelines' => 'events#guidelines', as: :guidelines_notifications
```

Comment:
Let's clean up the routes here. I prefer minimal routes when possible so the commented version on line 36 would be ideal if you can get it to work.  I also prefer to have the get route listed before the update one since you need to visit the show page before you can perform the update action.